build/build.cmd

```diff
@@ -13,4 +13,9 @@ go-licenses report github.com/microsoft/go-sqlcmd/cmd/modern --template build\NO
 copy %~dp0NOTICE.header + %~dp0notice.txt %~dp0..\NOTICE.md
 del %~dp0notice.txt
 
+REM Generates all versions of sqlcmd in platform-specific folder
+setlocal
+
+for /F "tokens=1-3 delims=," %%i in (%~dp0arch.txt) do set GOOS=%%i&set GOARCH=%%j&go build -o %~dp0..\%%i-%%j\%%k -ldflags="-X main.version=%sqlcmdVersion%" %~dp0..\cmd\modern
```

Comment:
To be able to generate all builds locally is fantastic! 
It has however increased the build time considerably. 
Can we wrap this logic under some command line argument so that we generate all the builds only if the argument is passed or can we introduce optional argument to generate build for current GOOS,GOARCH only?

Reply:
you could edit arch.txt or build.cmd locally and just not commit it.
`go build cmd/modern` is a fine way to build locally quickly too.
Now that we have different functionality depending on architecture I want to enforce people checking they don't break the multiplatform build in the default scenario, when they are generating the NOTICE etc.

Reply:
Indeed. Makes sense.